core/src/main/scala/org/labrad/Server.scala

```diff
@@ -360,47 +361,32 @@ object ServerConfig {
   }
 
   /**
-   * Create ServerConfig from command line and map of environment variables.
-   *
-   * @param args command line parameters
-   * @param env map of environment variables, which defaults to the actual
-   *        environment variables in scala.sys.env
-   * @return a Try containing a ServerArgs instance (on success) or a Failure
-   *         in the case something went wrong. The Failure will contain an
-   *         ArgotUsageException if the command line parsing failed or the
-   *         -h or --help options were supplied.
+   * Command line options for creating ServerConfig.
    */
-  def fromCommandLine(
-    args: Array[String],
-    env: Map[String, String] = scala.sys.env
-  ): Try[ServerConfig] = {
-    val parser = new ArgotParser("labrad-server",
-      preUsage = Some("Run a labrad server."),
-      sortUsage = false
-    )
-    val nameOpt = parser.option[String](
+  trait Options { _: Command =>
+    val name = parser.option[String](
       names = List("name"),
       valueName = "string",
       description = "The name to use for this server when connecting to the " +
         "labrad manager. If given, this will override the default name. " +
         "This is generally not needed, but can be useful in testing if you " +
         "want to run several copies of a given server without name conflicts."
```

Comment:
Yes, we should go with 100 chars which is the standard for java as well. I'm guessing scalafmt won't touch this sort of thing with concatenated string literals, but if it does that would be cool. Otherwise, we can manually reflow it at some point.